diff -uNr pytorch/torch/csrc/autograd/python_function.cpp pytorch.mod/torch/csrc/autograd/python_function.cpp
--- pytorch/torch/csrc/autograd/python_function.cpp	2022-10-03 21:59:02.604422931 +0900
+++ pytorch.mod/torch/csrc/autograd/python_function.cpp	2022-10-03 23:39:51.207871384 +0900
@@ -581,7 +581,7 @@
           throw torch::TypeError(
               "save_for_backward can only save variables, but argument %zu is of "
               "type %s",
-              i,
+              (long)(i),
               Py_TYPE(obj)->tp_name);
         }
       }
diff -uNr pytorch/torch/csrc/utils/python_arg_parser.h pytorch.mod/torch/csrc/utils/python_arg_parser.h
--- pytorch/torch/csrc/utils/python_arg_parser.h	2022-10-03 19:00:05.000000000 +0900
+++ pytorch.mod/torch/csrc/utils/python_arg_parser.h	2022-10-03 23:39:46.263853740 +0900
@@ -524,7 +524,7 @@
       "%s(): argument '%s' failed to unpack the object at pos %zu with error \"%s\"",
       args->signature.name.c_str(),
       args->signature.params[i].name.c_str(),
-      idx + 1,
+      (long)(idx + 1),
       error.c_str());
 }
 
@@ -707,7 +707,7 @@
           signature.params[i].name.c_str(),
           signature.params[i].type_name().c_str(),
           Py_TYPE(obj)->tp_name,
-          idx + 1);
+          (long)(idx + 1));
     }
   }
   return res;
